fixtures/flows/briefResponse/training.js

```diff
@@ -1,22 +1,28 @@
-export const respond = async (params) => {
-    console.log('Starting to respond to atm brief');
+import * as util from '../../flows/utils'
 
-    await clickSubmitResponse();
-    await matchText('a', 'Enter a date for when you can start the project');
-    await matchText('a', 'Choose a file for your written proposal');
-    await matchText('a', 'Choose a file for your project costs');
-    await matchText('a', 'Choose a file for your trainer résumés');
-    await matchText('a', 'A contact number is required');
+/* eslint-disable no-unused-vars */
```

Comment:
is this still needed? if so can it be put closer to where the line is with `eslint-disable-next-line`